frontend/components/MainPageV1/Home/index.tsx

```diff
@@ -109,8 +110,14 @@ export const Home = () => {
     ],
   );
 
+  // Scroll to top when selected agent type changes
+  useEffect(() => {
+    const scrollableParent = containerRef.current?.parentElement;
+    scrollableParent?.scrollTo({ top: 0, behavior: 'smooth' });
+  }, [selectedAgentType]);
+
```

Comment:
good addition! I think we need to do it on each page... when I click agent wallet - I have to scroll to bottom, and the wallet page is opened being scrolled to bottom as well

Reply:
moved it to the mainPage component, now it should work for all the "pages" rendered through the main page.